src/act_obj.hpp

```diff
@@ -16,3 +16,4 @@ bool obj_move_violates_uniqueness(Char *source_char, Char *dest_char, Object *mo
 bool obj_move_violates_uniqueness(Char *source_char, Char *dest_char, Object *moving_obj,
                                   GenericList<Object *> &objs_to);
 bool check_material_vulnerability(Char *ch, Object *object);
+void handle_corpse_summoner(Char *ch, Char *victim, Object *obj);
```

Comment:
This seems fine although it crossed my mind that we could also move it to CorpseSummoner.hpp if only for consistency (I'm happy to look at that separately).